### DISCUSSION TITLE
In-app audio equalizer

### DISCUSSION THREAD

**Discussion by Naozumi520** [ppy/osu#17066] [Ideas] [answered]
It's nice if we can have a equalizer in the game like we can use low-pass and high-pass filters to different headphones.
I know this can be done with a system-wide eq, but not everyone wants to apply equalizers to gobals.
Is this possible that we can have a in game equalizer?

**Comment by peppy** [comment, marked as answer]
This is out of scope sorry. I'd recommend using a system equaliser if you need one for your setup.

**Reply by AuracleTech** [reply]
Would have been cool, the system can't compensate for the audio latency added but the game could ☹

**Reply by peppy** [reply]
You can adjust the universal offset. That's the best the game could do for any added latency.
